openpathsampling/tests/test_spring_shooting.py

```diff
@@ -292,6 +297,15 @@ def test_correct_loading():
         sel.restart_from_step(step)
         assert sel.trial_snapshot == 10
 
+    def test_deprectation(self):
```

Comment:
```suggestion
    def test_deprecation(self):
```

(spelling)

(And why test the deprecation instead of making to not issue the deprecation?)